scripts/deploy.js

```diff
@@ -0,0 +1,98 @@
+const { Web3Connection, ERC20, NetworkFactory, Network } = require("@taikai/dappkit");
+
+const argv = require('yargs')
+    .alias('e', 'envFile')
+    .describe('e', 'Environment file to load')
+    .default('e', '.env')
+    .alias('n', 'network')
+    .describe('n', 'Ethereum Blockchain')
+    .default('n', 'development')
+    .alias('h', 'help')
+    .help('help')
+    .usage('Usage: $0 [options] <file>')
+    .showHelpOnFail(false, 'Specify --help for available options')
+    .epilog('TAIKAI Copyright 2022').argv;
+
+require('dotenv').config({ path: argv.envFile });
+
+const networks = {
+  development: {
+    url: 'http://127.0.0.1:8545/',
+  }, 
+  moonbase: {
+    url: 'https://rpc.api.moonbase.moonbeam.network',
+  },
+  moonbeam: {
+    url: 'https://rpc.api.moonbeam.network',
+  },    
+  kovan: {
+    url: `https://kovan.infura.io/v3/${process.env.DEPLOY_INFURA_KEY}`
+  },
+  mainnet: {
+    url: `https://mainnet.infura.io/v3/${process.env.DEPLOY_INFURA_KEY}`
+  },
+  ropsten: {
+    url: `https://ropsten.infura.io/v3/${process.env.DEPLOY_INFURA_KEY}`
+  },
+  seneca: {
+    url: 'http://eth-seneca.taikai.network:8545/',
+  }
+}
+
+async function main() {
+
+  const ownerAddress = process.env.DEPLOY_OWNER_ADDRESS;
```

Comment:
would be cool to have these (`DEPLOY_OWNER_ADDRESS` and `DEPLOY_PRIVATE_KEY`) as an argument as well, as we would concentrate every option on the entry command ^^

Reply:
yes i can do it.